testApi/src/main/scala/africa/absa/testing/scapi/rest/response/Response.scala

```diff
@@ -17,8 +17,9 @@
 package africa.absa.testing.scapi.rest.response
 
 import africa.absa.testing.scapi.json.ResponseAction
+import africa.absa.testing.scapi.logging.Logger
 
-case class Response(statusCode: Int, body: String, headers: Map[String, Seq[String]])
+case class Response(statusCode: Int, body: String, url: String, statusMessage: String, headers: Map[String, Seq[String]], cookies: Map[String, (String, Boolean)], duration: Long)
```

Comment:
Imho the cookie type `Map[String, (String, Boolean)]` does not communicate the intention well; then you have to do abysmal calls like:
`response.cookies(cookieName)._1` where you would rather want to do `response.cookies(cookieName).value`

Therefore: 
```suggestion
case class CookieValue(value: String, secured: Boolean)
case class Response(statusCode: Int, body: String, url: String, statusMessage: String, headers: Map[String, Seq[String]], cookies: Map[String, CookieValue], duration: Long)
```

Reply:
Fixed in commit 11293dcc6b1762816cfcd832e4fa189b42882327 and one before.